### PR TITLE
HBASE-27669 chaos-daemon.sh should make use hbase script start/stop c…

### PR DESCRIPTION
**Pull request by chrajeshbabu**
…haosagent and chaos monkey runner

### CODE REVIEW

**Comment by chrajeshbabu** [comment]
I have tested the scripts and able to start/stop chaosagent and chaosmonkeyrunner in the background.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 28s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 50s |  master passed  |
| +1 :green_heart: |  compile  |   4m 46s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m 14s |  master passed  |
| +1 :green_heart: |  spotless  |   0m 42s |  branch has no errors when running spotless:check.  |
| +1 :green_heart: |  spotbugs  |   7m 23s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 52s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 46s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 46s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m  2s |  the patch passed  |
| -0 :warning: |  shellcheck  |   0m  1s |  The patch generated 31 new + 39 unchanged - 27 fixed = 70 total (was 66)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  13m 59s |  Patch does not cause any errors with Hadoop 3.2.4 3.3.4.  |
| +1 :green_heart: |  spotless  |   0m 40s |  patch has no errors when running spotless:check.  |
| +1 :green_heart: |  spotbugs  |   7m 40s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 17s |  The patch does not generate ASF License warnings.  |
|  |   |  59m 35s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | dupname asflicense spotless shellcheck shelldocs javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 504cec87ce1e 5.4.0-1092-aws #100~18.04.2-Ubuntu SMP Tue Nov 29 08:39:52 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Eclipse Adoptium-11.0.17+8 |
| shellcheck | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| Max. process+thread count | 177 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/console |
| versions | git=2.34.1 maven=3.8.6 shellcheck=0.8.0 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 57s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 18s |  master passed  |
| +1 :green_heart: |  compile  |   1m 56s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 29s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m  6s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 17s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 51s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 51s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 27s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 268m 37s |  root in the patch passed.  |
|  |   | 300m 59s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 5266449020c7 5.4.0-135-generic #152-Ubuntu SMP Wed Nov 23 20:19:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Eclipse Adoptium-11.0.17+8 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/testReport/ |
| Max. process+thread count | 4611 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 11s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   2m 56s |  master passed  |
| +1 :green_heart: |  compile  |   1m 32s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 15s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 34s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 45s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 31s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 31s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 13s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 36s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 345m  0s |  root in the patch passed.  |
|  |   | 374m 26s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux c0d584a4bc09 5.4.0-137-generic #154-Ubuntu SMP Thu Jan 5 17:03:22 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Temurin-1.8.0_352-b08 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/testReport/ |
| Max. process+thread count | 4802 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/1/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 24s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 50s |  master passed  |
| +1 :green_heart: |  compile  |   4m 45s |  master passed  |
| +1 :green_heart: |  checkstyle  |   1m  6s |  master passed  |
| +1 :green_heart: |  spotless  |   0m 43s |  branch has no errors when running spotless:check.  |
| +1 :green_heart: |  spotbugs  |   7m 22s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   4m 44s |  the patch passed  |
| +1 :green_heart: |  javac  |   4m 44s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   1m  3s |  the patch passed  |
| -0 :warning: |  shellcheck  |   0m  1s |  The patch generated 7 new + 64 unchanged - 2 fixed = 71 total (was 66)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  hadoopcheck  |  13m 44s |  Patch does not cause any errors with Hadoop 3.2.4 3.3.4.  |
| +1 :green_heart: |  spotless  |   0m 41s |  patch has no errors when running spotless:check.  |
| +1 :green_heart: |  spotbugs  |   7m 32s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 18s |  The patch does not generate ASF License warnings.  |
|  |   |  58m 28s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | dupname asflicense spotless shellcheck shelldocs javac spotbugs hadoopcheck hbaseanti checkstyle compile |
| uname | Linux 7a9fa316caf2 5.4.0-1092-aws #100~18.04.2-Ubuntu SMP Tue Nov 29 08:39:52 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Eclipse Adoptium-11.0.17+8 |
| shellcheck | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| Max. process+thread count | 177 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/console |
| versions | git=2.34.1 maven=3.8.6 shellcheck=0.8.0 spotbugs=4.7.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  9s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  2s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 10s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 34s |  master passed  |
| +1 :green_heart: |  compile  |   1m 53s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 27s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m  5s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 21s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 58s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 58s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 19s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m  5s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 267m 54s |  root in the patch passed.  |
|  |   | 299m 41s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 95434e50668f 5.4.0-135-generic #152-Ubuntu SMP Wed Nov 23 20:19:22 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Eclipse Adoptium-11.0.17+8 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/testReport/ |
| Max. process+thread count | 4631 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 19s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ master Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   2m 46s |  master passed  |
| +1 :green_heart: |  compile  |   1m 32s |  master passed  |
| +1 :green_heart: |  shadedjars  |   4m 13s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 32s |  master passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 11s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   2m 43s |  the patch passed  |
| +1 :green_heart: |  compile  |   1m 34s |  the patch passed  |
| +1 :green_heart: |  javac  |   1m 34s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   4m 16s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   1m 34s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 344m  5s |  root in the patch passed.  |
|  |   | 371m 25s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.42 ServerAPI=1.42 base: https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/5078 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 040c42ab55d1 5.4.0-137-generic #154-Ubuntu SMP Thu Jan 5 17:03:22 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | master / d2b0074f7a |
| Default Java | Temurin-1.8.0_352-b08 |
|  Test Results | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/testReport/ |
| Max. process+thread count | 4895 (vs. ulimit of 30000) |
| modules | C: hbase-it . U: . |
| Console output | https://ci-hbase.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-5078/2/console |
| versions | git=2.34.1 maven=3.8.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java [hunk 1]
```diff
@@ -217,7 +217,7 @@ protected String findPidCommand(ServiceType service) {
     }
 
     public String signalCommand(ServiceType service, String signal) {
-      return String.format("%s | xargs sudo kill -s %s", findPidCommand(service), signal);
+      return String.format("%s | xargs kill -s %s", findPidCommand(service), signal);
```
Comment (Apache9) [inline hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java:220]: Why remove sudo?

Comment (chrajeshbabu) [reply hbase-it/src/test/java/org/apache/hadoop/hbase/HBaseClusterManager.java:220]: find pid command using ps ux which lists the processes owned by the user who is running the command. So to kill user owned processes sudo is not required hence removed.

**Pull request merged**